ERREUR dans : get_day_bt - 2024-07-29 21:39:03.200738
Erreur de type : <class 'AttributeError'>
Agrument de l'erreur : ("'NoneType' object has no attribute 'filter'",)
Description de l'erreur : 'NoneType' object has no attribute 'filter'
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]
 - jour : 10-10-2024ERREUR dans : get_day - 2024-07-29 21:39:03.679735
Erreur de type : <class 'interactions.client.errors.NotFound'>
Agrument de l'erreur : ('404|Not Found: (10062) Unknown interaction',)
Description de l'erreur : HTTPException: 404|Not Found || Unknown interaction
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]
 - jour : 10-10-2024ERREUR dans : info - 2024-07-29 21:43:59.904488
Erreur de type : <class 'AttributeError'>
Agrument de l'erreur : ("'NoneType' object has no attribute 'filiere'",)
Description de l'erreur : 'NoneType' object has no attribute 'filiere'
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1267397506518483045, name='error-log', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]ERREUR dans : info - 2024-07-29 21:44:44.644491
Erreur de type : <class 'AttributeError'>
Agrument de l'erreur : ("'NoneType' object has no attribute 'value'",)
Description de l'erreur : 'NoneType' object has no attribute 'value'
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]ERREUR dans : info - 2024-07-29 21:48:55.636560
Erreur de type : <class 'TypeError'>
Agrument de l'erreur : ("'function' object is not iterable",)
Description de l'erreur : 'function' object is not iterable
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]ERREUR dans : info - 2024-07-29 21:52:25.733299
Erreur de type : <class 'TypeError'>
Agrument de l'erreur : ("'function' object is not iterable",)
Description de l'erreur : 'function' object is not iterable
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False)]ERREUR dans : ics - 2024-07-30 10:41:47.461258
Erreur de type : <class 'KeyboardInterrupt'>
Agrument de l'erreur : ()
Description de l'erreur : 
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]ERREUR dans : ics - 2024-07-30 10:43:35.700992
Erreur de type : <class 'KeyboardInterrupt'>
Agrument de l'erreur : ()
Description de l'erreur : 
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265996035357147136, name='TD 3 Inge Anglais', position=11, purchasable_or_has_subscribers=False)]ERREUR dans : today - 2024-07-30 10:43:50.752192
Erreur de type : <class 'TypeError'>
Agrument de l'erreur : ("Event.__init__() missing 1 required positional argument: 'uid'",)
Description de l'erreur : Event.__init__() missing 1 required positional argument: 'uid'
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]ERREUR dans : today - 2024-07-30 10:43:54.777698
Erreur de type : <class 'TypeError'>
Agrument de l'erreur : ("Event.__init__() missing 1 required positional argument: 'uid'",)
Description de l'erreur : Event.__init__() missing 1 required positional argument: 'uid'
Les paramètres de la fonction étais : 
 - auteur : @kaawan
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265996035357147136, name='TD 3 Inge Anglais', position=11, purchasable_or_has_subscribers=False)]ERREUR dans : today - 2024-07-30 10:45:51.406768
Erreur de type : <class 'RecursionError'>
Agrument de l'erreur : ('maximum recursion depth exceeded',)
Description de l'erreur : maximum recursion depth exceeded
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]ERREUR dans : ics - 2024-07-30 10:51:21.257279
Erreur de type : <class 'KeyboardInterrupt'>
Agrument de l'erreur : ()
Description de l'erreur : 
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]ERREUR dans : ics - 2024-07-30 10:53:08.693804
Erreur de type : <class 'interactions.client.errors.NotFound'>
Agrument de l'erreur : ('404|Not Found: (10062) Unknown interaction',)
Description de l'erreur : HTTPException: 404|Not Found || Unknown interaction
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]ERREUR dans : get_day_bt - 2024-07-30 15:40:39.464199
Erreur de type : <class 'AttributeError'>
Agrument de l'erreur : ("'dict_values' object has no attribute 'copy'",)
Description de l'erreur : 'dict_values' object has no attribute 'copy'
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]
 - jour : 04-09-2024ERREUR dans : get_day_bt - 2024-07-30 15:41:32.338317
Erreur de type : <class 'AttributeError'>
Agrument de l'erreur : ("'dict_values' object has no attribute 'copy'",)
Description de l'erreur : 'dict_values' object has no attribute 'copy'
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]
 - jour : 04-09-2024ERREUR dans : get_week_bt - 2024-07-30 15:46:26.067230
Erreur de type : <class 'interactions.client.errors.HTTPException'>
Agrument de l'erreur : ('400|Bad Request: (50035) Invalid Form Body',)
Description de l'erreur : HTTPException: 400|Bad Request: data->embeds BASE_TYPE_MAX_LENGTH: Must be 10 or fewer in length.
Les paramètres de la fonction étais : 
 - auteur : @dany_d45
 - serveur :  Guild(id=1265571990052274207, name='Serveur de test', description=None)
 - message :  None
 - channel :  GuildText(id=1265572459286106183, name='test-bot', type=<ChannelType.GUILD_TEXT: 0>)
 - role member :  [Role(id=1265572644095393833, name='Admin', position=23, purchasable_or_has_subscribers=False), Role(id=1265572852623605790, name='Modérateur', position=22, purchasable_or_has_subscribers=False), Role(id=1265573289443590164, name='Ingé', position=20, purchasable_or_has_subscribers=False), Role(id=1265995794235003011, name='TD 2 Inge', position=18, purchasable_or_has_subscribers=False), Role(id=1265994595460976690, name='TP C Inge', position=15, purchasable_or_has_subscribers=False), Role(id=1265995941253746738, name='TD 1 Inge Anglais', position=13, purchasable_or_has_subscribers=False), Role(id=1265996730697121964, name='TP C Miage', position=5, purchasable_or_has_subscribers=False)]
 - semaine : 10-10-2024